src/index.html

```diff
@@ -17,16 +17,44 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      method="get"
+      class="search-bar search bar--big"
+      data-qa="big"
+    >
+      <label
+        class="search-bar__label search-bar_label--big"
+        for="search-bar search-bar_label--big"
```

Comment:
The 'for' attribute should not contain spaces. Consider using a single identifier or using hyphens/underscores to separate words.